lib/rucio/api/authorisation.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2012-2018 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Mario Lassnig <mario@lassnig.net>, 2018
+
+from rucio.api import permission
+from rucio.common import exception
+from rucio.core import authorisation
+
+
+def get_signed_url(account, appid, ip=None, service=None, operation=None, url=None):
```

Comment:
`rse`, 'scheme' and 'impl' in the signature ? This plays on how are accessed and stored the credentials (access_keys etc) plus the permission and monitoring. A bit of thought is needed regarding service vs RSE